docs/source/install/upgrades.rst

```diff
@@ -167,6 +167,102 @@ The following sections call out the migration scripts that need to be run when u
 respective version. If you are upgrading across multiple versions, make sure you run the scripts for
 any skipped versions:
 
+
+v3.3
+''''
+
+* MongoDB 4.0 is the new default version. Previously the version of MongoDB was 3.4 and
```

Comment:
The version was only 3.4 before for U16 and EL7. For EL8 and U18 it was already MongoDB 4.0, so the upgrade is only required for some OSes.
Can we add something to state it only applies to those OSes?